tester_app/client/app/bundles/HelloWorld/constants/helloWorldConstants.jsx

```diff
@@ -0,0 +1,5 @@
+import mirrorCreator from 'mirror-creator';
```

Comment:
Place comment above, editing for readability:

// See https://www.npmjs.com/package/mirror-creator
// Allows us to easily setup constants in HelloWorldActionCreators.jsx:
//  import * as actionTypes from '../constants/helloWorldConstants';
//  export function updateName(name) {
//  return { 
//    type: actionTypes.HELLO_WORLD_NAME_UPDATE,
//    name,
//  };
//}